src/serverenvironment.cpp

```diff
@@ -1882,16 +1882,42 @@ void ServerEnvironment::getSelectedActiveObjects(
 */
 
 u16 ServerEnvironment::addActiveObjectRaw(std::unique_ptr<ServerActiveObject> object_u,
-	bool set_changed, u32 dtime_s)
+	const StaticObject *from_static, u32 dtime_s)
 {
 	auto object = object_u.get();
 	if (!m_ao_manager.registerObject(std::move(object_u))) {
 		return 0;
 	}
 
+	// Register reference in scripting api (must be done before post-init)
+	m_script->addObjectReference(object);
+	// Post-initialize object
+	// Note that this can change the value of isStaticAllowed() in case of LuaEntitySAO
+	object->addedToEnvironment(dtime_s);
+
+	// Activate object
+	if (object->m_static_exists)
+	{
```

Comment:
wrong brace :)

Reply:
intentional so I can properly place the comment before `else`, suggestions welcome